app/mattermost_managed/mattermost-managed.android.js

```diff
@@ -35,7 +37,7 @@ export default {
         }
     },
     authenticate: LocalAuth.auth,
-    blurAppScreen: MattermostManaged.blurAppScreen,
+    blurAppScreen: emptyFunction,
```

Comment:
so on android we don't need [this](https://github.com/mattermost/mattermost-mobile/blob/1efb01deda81cc0ee0fa0a36af8c02e342272f11/app/init/emm_provider.js#L121) anymore I see